### PR TITLE
Docs: Added weekly update for June 17

### PR DESCRIPTION
**Pull request by ponori**

### CODE REVIEW

File: docs/pages/BlogPosts.json [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
   "digests": [
+    {
+      "week": "June 17, 2022",
+      "posts": [
+        {
+          "title": "Dark mode Figma variants, Phase 1",
+          "audience": ["Design"],
+          "imageSrc": "https://i.pinimg.com/originals/90/d6/86/90d6869063bd80e56fe0820eebf10aca.png",
+          "imageAltText": "A dark mode variant of Popover",
+          "content": "This has been a long time coming, but we've shipped our first phase of dark mode variants in our Gestalt libraries! This first phase has focused on core components for our [Gestalt for iOS library](https://www.figma.com/file/AHcKJDgb7E7YswlgW1wY8E/Gestalt-for-iOS) (e.g., Button, Avatar, Popover, etc.). Our next phase will be to offer dark mode for more complex components like Pin, Board, and other reps. We look forward to sharing many more updates _soon_."
```
Comment (dangerismycat) [inline docs/pages/BlogPosts.json:12]: you may want to clarify that this is "Gestalt _Figma_ libraries"

File: docs/pages/BlogPosts.json [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+          "content": "We added [typography guidelines](https://gestalt.pinterest.systems/typography) a couple weeks ago and now we're coming in hot with design guidelines for [Text](https://gestalt.pinterest.systems/text) and [Heading](https://gestalt.pinterest.systems/heading). Our goal for these updates is to develop a strong foundation of typographic guidance that we can build upon in future releases."
+        },
+        {
+          "title": "Gestalt Learning released",
+          "audience": ["Engineering", "Design"],
+          "imageSrc": "https://i.pinimg.com/originals/13/65/cd/1365cddf722bf00aa21a21e88c1eba36.png",
+          "imageAltText": "Screenshots of Gestalt Learning",
+          "content": "We know that getting up to speed with Gestalt can take some work and we're constantly working to improve the onboarding process. That's why we released [Gestalt Learning](http://pinch/gestaltLearning) to offer a self-service learning path for important Gestalt topics. We have thoughts for additional content, but we'd love to hear your ideas! So, please reach out and give us your requests. \n\n**Note:** This resource is available to Pinterest employess only."
```
Comment (dangerismycat) [inline docs/pages/BlogPosts.json:33]: typo: "employees"

**Review by dangerismycat — APPROVED** [review]

File: docs/pages/BlogPosts.json (hunk 1 above)
Comment (ponori) [reply docs/pages/BlogPosts.json:12]: Fixed!

File: docs/pages/BlogPosts.json (hunk 2 above)
Comment (ponori) [reply docs/pages/BlogPosts.json:33]: Fixed!

**Pull request merged**